packages/cli-e2e/__tests__/auth.specs.ts

```diff
@@ -23,20 +24,19 @@ describe('auth', () => {
 
     it('should open the platform page', async () => {
       // TODO CDX-98: Remove `-e=dev`.
-      const cliProcess = processManager.spawn(CLI_EXEC_PATH, [
-        'auth:login',
-        '-e=dev',
-      ]);
-      cliProcess.stderr.on('data', async (data) => {
-        if (isYesNoPrompt(data.toString())) {
-          await answerPrompt('n', cliProcess);
-        }
-      });
-      cliProcess.stdout.on('data', async (data) => {
-        if (isYesNoPrompt(data.toString())) {
-          await answerPrompt('n', cliProcess);
-        }
-      });
+      const cliTerminal = new Terminal(
+        CLI_EXEC_PATH,
+        ['auth:login', '-e=dev'],
+        undefined,
+        processManager,
+        'auth-login'
+      );
```

Comment:
To avoid telescoping constructors, can we move some of these optional parameters into a single option?

Reply:
I'm not sure to understand what you mean?
One thing I try to do tho is to avoid matryoshka arguments and keep the tests 'flat'. (i.e. avoid 42th level of indentation)

However, we certainly may do better processing of the args, I guess we could do that simply with the command chaining. I will do something akin to that in another PR ;)